docs/resources/azure_security_center_policies.md

```diff
@@ -0,0 +1,84 @@
+---
+title: About the azure_security_center_policies Resource
+platform: azure
+---
+
+# azure_security_center_policies
+
+Use the `azure_security_center_policies` InSpec audit resource to test properties and configuration of multiple Azure Policys.
```

Comment:
Typo `multiple Azure Policys` --> `multiple Azure Policies`

Reply:
This is fixed now.